spec/support/helpers/configuration_helper.rb

```diff
@@ -1,11 +1,15 @@
 # frozen_string_literal: true
 
+require 'dotenv'
+
+Dotenv.load('.env.test')
```

Comment:
Are you sure this is needed? I read the docs and it look like it would pick this up automatically if it exists.

Reply:
Correct. I did need it in my `bin/get_tokens`, but not here.

Reply:
Hmm, actually, I can't get it to work without explicitly loading `.env.test`...